doc/libportal.md

```diff
@@ -0,0 +1,14 @@
+Title: libportal Reference Manual
+
+# Portals
+
+libportal is a library that provides GIO-style async API wrappers
+for the D-Bus portal APIs in xdg-desktop-portal.
+
+Where possible, applications should prefer to use existing library
+APIs that have portal support. For example, a GTK+ application should
+use [class@Gtk.FileChooserNative] instead of [method@Portal.open_file].
+
+Some portal interfaces are not covered in libportal, since
+they already have perfectly adequate APIs in GLib (which
+libportal depends on anyway).
```

Comment:
The list of APIs was lost here.

Reply:
Yes. Not sure how to handle that, as the docs are all now in the Portal class.

Reply:
Well this section was just listing GLib API, which is valuable to keep:

```xml

      <variablelist>
        <varlistentry>
          <term>NetworkMonitor</term>
          <listitem><para>
            Use GNetworkMonitor
          </para></listitem>
        </varlistentry>
        <varlistentry>
          <term>ProxyResolver</term>
          <listitem><para>
            Use GProxyResolver
          </para></listitem>
        </varlistentry>
        <varlistentry>
          <term>Settings</term>
          <listitem><para>
            Use a plain GDBusProxy
          </para></listitem>
        </varlistentry>
        <varlistentry>
          <term>GameMode</term>
          <listitem><para>
            Not currently covered
          </para></listitem>
        </varlistentry>
        <varlistentry>
          <term>Secrets</term>
          <listitem><para>
            Not currently covered
          </para></listitem>
        </varlistentry>
      </variablelist>
```

Reply:
Does
``` 
Some portal interfaces are not covered in libportal, since
they already have perfectly adequate APIs in GLib (which libportal depends on anyway).

- NetworkMonitor: Use [iface@Gio.NetworkMonitor]
- ProxyResolver: Use [iface@Gio.ProxyResolver]
- Settings: Use a plain [class@Gio.DBusProxy]
- GameMode: Not currently covered
- Secrets: Not currently covered
```
work? I added that to the docs.

Reply:
Seems good.